controllers/front/OrderDetailController.php

```diff
@@ -108,17 +108,31 @@ public function postProcess()
                     }
 
                     if (Validate::isLoadedObject($customer)) {
-                        Mail::Send($this->context->language->id, 'order_customer_comment', Mail::l('Message from a customer'),
-                        array(
-                            '{lastname}' => $customer->lastname,
-                            '{firstname}' => $customer->firstname,
-                            '{email}' => $customer->email,
-                            '{id_order}' => (int)$order->id,
-                            '{order_name}' => $order->getUniqReference(),
-                            '{message}' => Tools::nl2br($msgText),
-                            '{product_name}' => $product_name
-                        ),
-                        $to, $toName, $customer->email, $customer->firstname.' '.$customer->lastname);
+                        Mail::Send(
+                            $this->context->language->id,
+                            'order_customer_comment',
+                            Mail::l('Message from a customer'),
+                            array(
+                                '{lastname}' => $customer->lastname,
+                                '{firstname}' => $customer->firstname,
+                                '{email}' => $customer->email,
+                                '{id_order}' => (int)$order->id,
+                                '{order_name}' => $order->getUniqReference(),
+                                '{message}' => Tools::nl2br($msgText),
+                                '{product_name}' => $product_name
+                            ),
+                            $to,
+                            $toName,
+                            strval(Configuration::get('PS_SHOP_EMAIL')),
```

Comment:
@basma-yangui-prestashop As you can see here, the `$from` email is the shop email. So I'm not sure what the issue you pointed out is about. For reference, here is the code comment block of `Mail::send()`:


     /**
     * Send Email
     *
     * @param int $id_lang Language ID of the email (to translate the template)
     * @param string $template Template: the name of template not be a var but a string !
     * @param string $subject Subject of the email
     * @param string $template_vars Template variables for the email
     * @param string $to To email
     * @param string $to_name To name
     * @param string $from From email
     * @param string $from_name To email
     * @param array $file_attachment Array with three parameters (content, mime and name). You can use an array of array to attach multiple files
     * @param bool $mode_smtp SMTP mode (deprecated)
     * @param string $template_path Template path
     * @param bool $die Die after error
     * @param int $id_shop Shop ID
     * @param string $bcc Bcc recipient (email address)
     * @param string $reply_to Email address for setting the Reply-To header
     * @return bool|int Whether sending was successful. If not at all, false, otherwise amount of recipients succeeded.
     */